src/textual/await_remove.py

```diff
@@ -5,7 +5,11 @@
 
 
 class AwaitRemove:
-    """An awaitable returned by App.remove and DOMQuery.remove."""
+    """An awaitable returned by a method that removes DOM nodes.
+
+    Returned by [Widget.remove][textual.widget.Widget.remove] and
```

Comment:
Instead of listing categorically the places where this thing can be returned from, might I suggest we rephrase it as "returned, for example, from ..." or "[...] by ..."?
The rationale is to make it less likely that the docstring will become wrong in the future.

Reply:
Funnily enough I had more or less that to start with and then decided against it as I felt this was one time where I'd want this to be categorically correct (in that we shouldn't really want more than those instances of `remove` in Textual).